### ISSUE TITLE
The `get_first` lint is applied to VecDeque but the help only mentions `first()`and not `front()`

### ISSUE THREAD

**Issue by HEnquist** [rust-lang/rust-clippy#12063]
### Description

The help text for the `get_first` lint assumes that `first()` is the method to use instead of `get(0)`, but this does not exist for a VecDeque. There, the `front()` method is the equivalent.
The text is defined here:
https://github.com/rust-lang/rust-clippy/blob/7f185bdef6791f0f4c7e89cbdc881964a58c9414/clippy_lints/src/methods/mod.rs#L2596
I'm assuming that the text was written before the lint was applied also to VecDeque, but now it's a bit misleading.

The suggestion however is correct, example:
`help: try: `self.buffer.front()`

**Comment by samueltardieu** [comment]
Thanks for your report. Could you provide a complete reproducer? The code below:

```rust
fn main() {
    let v = std::collections::VecDeque::from_iter([0]);
    v.get(0).unwrap();
}
```

produces the proper warning with Clippy's master branch:

```
warning: accessing first element with `v.get(0)`
 --> /tmp/t.rs:3:5
  |
3 |     v.get(0).unwrap();
  |     ^^^^^^^^ help: try: `v.front()`
  |
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#get_first
  = note: `#[warn(clippy::get_first)]` on by default

warning: 1 warning emitted
```

**Comment by HEnquist** [comment]
Ah sorry my description was unclear, clippy gives the right suggestion! The issue is only in the linked web page: https://rust-lang.github.io/rust-clippy/master/index.html#get_first
